packages/web-twig/src/Resources/components/Breadcrumbs/README.md

```diff
@@ -105,5 +105,35 @@ You can add `id`, `data-*` or `aria-*` attributes to further extend the componen
 descriptiveness and accessibility. Also, UNSAFE styling props are available,
 see the [Escape hatches][escape-hatches] section in README to learn how and when to use them.
 
+## BreadcrumbsItem
+
+Use the `BreadcrumbsItem` component for the ordered list as the component's children instead of passing the breadcrumb items array via props:
+
+```twig
+<Breadcrumbs>
+  {% for item in items %}
+    <BreadcrumbsItem isCurrent={{ items.length === index - 1 }} href={{ item.url }}>
```

Comment:
packages/web-twig/src/Resources/components/Breadcrumbs/Breadcrumbs.twig Here you use single brackets to pass the props. Which is correct, please?

Reply:
Ok, fixed.